### PR TITLE
Faction updoots

### PR DESCRIPTION
**Pull request by skull132**
- rscadd: "All factions now have a unique ID card."
  - rscadd: "Added the independent faction. At the moment, NT still retains access to the Freelance Journalist job, because the job's system is literal ass and I need to wait for another PR before redoing it."
  - tweak: "Tweaked the faction selection menu. It now links to the wiki and all."
  - rscdel: "Removed faction specific gear, since it's already present in the job system."

### CODE REVIEW

File: code/game/jobs/faction/hephaestus.dm [hunk 1]
```diff
@@ -33,23 +57,29 @@
...
 	uniform = /obj/item/clothing/under/rank/hephaestus
+	id = /obj/item/weapon/card/id/hephaestus
 
 /datum/outfit/job/mining/hephaestus
 	name = "Miner - Hephaestus"
 	uniform = /obj/item/clothing/under/rank/hephaestus
+	id = /obj/item/weapon/card/id/hephaestus
 
 /datum/outfit/job/cargo_tech/hephaestus
 	name = "Cargo TEchnician - Hephaestus"
```
Comment (ParadoxSpace) [inline code/game/jobs/faction/hephaestus.dm]: Technician

Comment (skull132) [reply code/game/jobs/faction/hephaestus.dm]:
```suggestion
	name = "Cargo Technician - Hephaestus"
```

File: interface/interface.dm [hunk 2]
```diff
@@ -1,15 +1,20 @@
...
+	if(config.wikiurl)
 		if(alert("This will open the wiki in your browser. Are you sure?",,"Yes","No")=="No")
 			return
-		to_chat(src, link(config.wikiurl))
+
+		var/to_open = config.wikiurl
+		if (sub_page)
+			to_open += sub_page
+
+		to_chat(src, link(to_open))
```
Comment (Arrow768) [inline interface/interface.dm]:
```suggestion
		send_link(src, to_open)
```

Comment (skull132) [reply interface/interface.dm]: Done and fixed.

**Pull request merged**